soccer/Processor.cpp

```diff
@@ -640,20 +650,21 @@ void Processor::applyJoystickControls(const JoystickControlValues& controlVals,
     }
 
     // translation
-    tx->set_body_x(translation.x);
-    tx->set_body_y(translation.y);
+    tx->set_xvelocity(translation.x);
+    tx->set_yvelocity(translation.y);
 
     // rotation
-    tx->set_body_w(controlVals.rotation);
+    tx->set_avelocity(controlVals.rotation);
 
     // kick/chip
     bool kick = controlVals.kick || controlVals.chip;
-    tx->set_kick_immediate(kick);
-    tx->set_kick(kick ? controlVals.kickPower : 0);
-    tx->set_use_chipper(controlVals.chip);
+    tx->set_triggermode(Packet::Control::IMMEDIATE);
+    tx->set_kcstrength(kick ? controlVals.kickPower : 0);
```

Comment:
We probably don't need to check if `kick == true` here since it will only kick/chip if the triggermode is set to do so.